test/unit_plot.jl

```diff
@@ -88,108 +114,41 @@ for N in [Float64, Rational{Int}, Float32]
     plot(cp)
     plot(cpa)
 
-    # ε-close
-    ε = N(1e-2)
-    if N == Float64 # TODO see #578
-        plot(b1, ε)
-        plot(bi, ε)
-        plot(hr, ε)
-        plot(ls, ε)
-        @test_throws AssertionError plot(itv, ε) # TODO see #575
-        plot(st, ε)
-        plot(zt, ε)
-        plot(zs, ε)
-        plot(hpg, ε)
-        plot(hpgo, ε)
-        plot(hpt, ε)
-        plot(vpg, ε)
-        plot(vpt, ε)
-        plot(es, ε)
-        @test_throws ErrorException plot(hs, ε) # TODO see #576/#578
-        @test_throws ErrorException plot(hp, ε) # TODO see #576/#578
-        @test_throws ErrorException plot(l, ε) # TODO see #576/#578
-        plot(uni, ε)
-        plot(ch, ε)
-        plot(cha, ε)
-        plot(sih, ε)
-        plot(lm, ε)
-        plot(rm, ε)
-        plot(ms, ε)
-        plot(msa, ε)
-        plot(cms, ε)
-        plot(cp, ε)
-        plot(cpa, ε)
-    else
-        @test_throws ErrorException plot(b1, ε) # TODO see #578
-    end
+    # -----
+    # cases that are not implemented
+    # -----
+    @test_throws ErrorException plot(itsa) # TODO not implemented yet
 end
 
-# set types that do not work with Rational{Int}
-for N in [Float64, Float32]
-    v0 = zeros(N, 2)
+# -----------------------------------------------------------------
+# tests for rationals; iterative refinement is not available
+# -----------------------------------------------------------------
+for N in [Rational{Int}]
```

Comment:
I would rather merge this loop with the one above. Simply add all `Rational{Int}` tests to the top and then `continue` the loop if `N != Rational{Int}`.

Reply:
Good, i have merged them, it does simplify the code! Tests pass locally.

Thanks a lot for the review.